project-base/tests/ReadModelBundle/Functional/Twig/ImageExtensionTest.php

```diff
@@ -72,7 +80,7 @@ public function testGetImageHtml(): void
             $this->getCurrentUrl()
         );
         $expected .= sprintf(
-            '    <img alt="" class="image-product" itemprop="image" data-src="%s/content-test/images/product/default/1.jpg" title="" src="" loading="lazy"/>',
+            '    <img alt="" class="image-product" itemprop="image" data-src="%s/content-test/images/product/default/1.jpg" title="" src="%1$s/content-test/images/product/default/1.jpg" loading="lazy"/>',
```

Comment:
Test for browsers without native lazyload support would be nice :)

Reply:
I agree... I added simple test in 8f6f2a1a8. Do you think it's ok that way?